website/netlify.toml

```diff
@@ -0,0 +1,29 @@
+[[redirects]]
+  from = "https://nickel-lang.netlify.com/*"
+  to = "https://nickel-lang.org/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "http://nickel-lang.netlify.com/*"
+  to = "http://nickel-lang.org/:splat"
```

Comment:
Is it on purpose that the target is `http` and not `https`?

Reply:
Fixed. Good catch!